libtiledbsoma/src/reindexer/reindexer.h

```diff
@@ -96,6 +96,10 @@ class IntIndexer {
      * Number of elements in the map set by map_locations
      */
     int map_size_ = 0;
+    /*
+     * Number of requested threads
+     */
+    int thread_count_ = 0;
```

Comment:
this new value seems redundant with tiledb_thread_pool, at least as currently used

Reply:
Good catch